src/types/global.d.ts

```diff
@@ -20,6 +20,7 @@ declare global {
     | `${PixelValue} ${PixelValue} ${PixelValue} ${PixelValue}`;
   type Padding = Margin;
   type User = {
+    readAt: any;
```

Comment:
type을 `any` 보다는 구체적으로 명시하는 것이 좋을 것 같습니다 !

```suggestion
    readAt?: Date;
```